app/controllers/searches_controller.rb

```diff
@@ -6,8 +6,9 @@ def show
     druid = params.require(:q)
     druid = "druid:#{druid}" unless druid.start_with?('druid:')
     result = Work.find_by(druid: druid) || Collection.find_by(druid: druid)
-    return render js: "window.location='#{polymorphic_path(result)}'" if result
+    return render json: { message: 'not found' }, status: :not_found unless result
```

Comment:
Can we remove this whole controller?

Reply:
Do we still want it to do a popup if the druid is not found or go to the same result you get from the "search for druid" page?

Reply:
I went ahead and made this box behave like the other one for consistency, using the same controller as you suggested.